docker/openproblems-r-base/README.md

```diff
@@ -23,7 +23,6 @@ R packages:
 
 * BiocManager
 * scran
-* IRKernel
```

Comment:
here as well... i think we may need to keep this for scib to work (or any calling of scran normalization via rpy2). Does scprep not also run via rpy2?

Reply:
IRKernel is not required for rpy2